gui/services/migrations/0018_add_asigra_service.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import unicode_literals
+from django.db import migrations
+
+import sys
+
+
+def add_asigra(apps, schema_editor):
```

Comment:
Service entry should only be created for TrueNAS, so it doesn't show in FreeNAS API calls.